src/components/login.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import { AuthorizationContext } from "../authorizationProvider";
+import { Form } from "react-final-form";
+import FormTextField from "../components/common/formTextField"
+
+class Login extends Component {
+
+  onSubmit = async values => {
+    await this.context.login(values);
```

Comment:
tu by to malo byt tiez obalene v try/catch - ak sa uspesne clovek prihlasi, moze nastat redirect niekam, napr. na hlavnu stranku (history.push). ak nastane chyba, zobrazit notifikaciu.

Reply:
error handling je vec na ktoru potrebujem trosku casu ale urcite sa na to chystam.. momentalne to nie je ani zdaleka v dobrom stave